packages/vuetify/src/labs/VTreeview/VTreeviewItem.tsx

```diff
@@ -0,0 +1,110 @@
+// Styles
+import './VTreeviewItem.sass'
+
+// Components
+import { VBtn } from '@/components/VBtn'
+import { VListItemAction } from '@/components/VList'
+import { makeVListItemProps, VListItem } from '@/components/VList/VListItem'
+
+// Composables
+import { IconValue } from '@/composables/icons'
+import { useLink } from '@/composables/router'
+
+// Utilities
+import { computed, inject, ref } from 'vue'
+import { genericComponent, propsFactory, useRender } from '@/util'
+
+// Types
+import { VTreeviewSymbol } from './VTreeview'
```

Comment:
This is a circular import: VTreeview -> VTreeviewChildren -> VTreeviewItem -> VTreeview